RELEASE/scripts/autoscend/auto_heavyrains.ash

```diff
@@ -588,8 +588,10 @@ boolean L13_towerFinalHeavyRains()
 	}
 	
 	//buff up before the boss
-	//buffMaintain($effect[Benetton's Medley of Diversity], 0, 1, 1);		//15 prismatic weapon dmg. TODO: implemented in buffMaintain
+	buffMaintain($effect[Benetton's Medley of Diversity], 0, 1, 1);			//15 prismatic weapon dmg.
 	buffMaintain($effect[Dirge of Dreadfulness], 0, 1, 1);					//12 spooky weapon dmg
+	buffMaintain($effect[Boner Battalion], 0, 1, 1);						//32-33 sleaze and spooky passive dmg
+	buffMaintain($effect[Frigidalmatian], 0, 1, 1);							//40 (due to cap) cold passive dmg
 	effectAblativeArmor(true);					//Unimportant effects protect your important one from being removed.
```

Comment:
In a future PR, you could consider a custom `addToMaximizer()` call that added terms for `spooky dmg max 50` and so on, with `-acc1, -acc2, -acc3, -hat, [etc]`. Then running the maximizer.

Reply:
oh, that is a good idea.